test/test_sparse_semi_structured.py

```diff
@@ -598,6 +598,8 @@ def setUp(self):
 
     @parametrize("dense_input_shape", [(128, 128)])
     def test_cslt_sparse_mm_int8_in_fp16_out(self, dense_input_shape, device):
+        assert False
```

Comment:
just to make sure it's getting built with cusparselt, will remove later

Reply:
You were building with CUSPARELT, see https://github.com/pytorch/pytorch/actions/runs/7215814225/job/19660823130#step:12:1169